microraiden/microraiden/client/http_client.py

```diff
@@ -135,3 +138,10 @@ def on_payment_requested(
             channel_manager_address: str
     ):
         return True
+
+    def on_http_response(self,
```

Comment:
Two notes:
* Since we're using type hints (albeit in a bit of a half-assed fashion), please include `reply: Response` with `from requests import Response` here.
* (sorry for nitpicking) using the Raiden style guide we should start multiline function calls on the next line, like so:
```python
def on_http_response(
    self,
    resource: str,
    reply: Response
):
```
or 
```python
def on_http_response(
    self,
    resource: str,
    reply: Response):
```

Reply:
Is there a tool to check/enforce type hints? My `mypy` integration is not really usable yet, I should spend some time making it work.
ad multine calls, I always forget them. I prefer less lines in the code so it's hard to keep it on mind if there's no linter to yell at me :-) It'll get better over the time.